core/templates/dev/head/pages/exploration_editor/history_tab/HistoryTab.js

```diff
@@ -53,6 +53,9 @@ oppia.controller('HistoryTab', [
     var explorationSnapshots = null;
     var versionTreeParents = null;
     var nodesData = null;
+    $scope.versionNumbersToDisplay = [];
+    $scope.currentPage = 1;
+    $scope.versionsPerPage = 30;
```

Comment:
Make this a constant: $scope.VERSIONS_PER_PAGE

Reply:
Done.